tests/e2e/specs/api/DevfileAcceptanceTestAPI.spec.ts

```diff
@@ -3,47 +3,102 @@ import { DevWorkspaceConfigurationHelper } from '../../utils/DevWorkspaceConfigu
+                        if (command.exec.group.kind === 'build') {
+                            Logger.debug(`Build command found: ${command.exec.commandLine}`);
+                            devfilesBuildCommands.push(command);
+                        }
+                    });
+                }
+            });
 
-        kubernetesCommandLineToolsExecutor = new KubernetesCommandLineToolsExecutor(devWorkspaceName);
-        kubernetesCommandLineToolsExecutor.loginToOcp();
-    });
+            test('Create DevWorkspace', async function (): Promise<void> {
+                const devWorkspaceConfigurationYamlString: string = await devWorkspaceConfigurationHelper.getDevWorkspaceConfigurationYamlAsString(devfileContext);
+                const applyOutput: ShellString = kubernetesCommandLineToolsExecutor.applyYamlConfigurationAsStringOutput(devWorkspaceConfigurationYamlString);
 
-    test('Create DevWorkspace', async function (): Promise<void> {
-        const devWorkspaceConfigurationYamlString: string = await devWorkspaceConfigurationHelper.getDevWorkspaceConfigurationYamlAsString(devfileContext);
-        const applyOutput: ShellString = kubernetesCommandLineToolsExecutor.applyYamlConfigurationAsStringOutput(devWorkspaceConfigurationYamlString);
+                expect(applyOutput.stdout)
+                    .contains('devworkspacetemplate')
+                    .and.contains('devworkspace')
+                    .and.contains.oneOf(['created', 'configured']);
 
-        expect(applyOutput.stdout)
-            .contains('devworkspacetemplate')
-            .and.contains('devworkspace')
-            .and.contains.oneOf(['created', 'configured']);
+            });
 
-    });
+            test('Wait until DevWorkspace has status "ready"', async function (): Promise<void> {
+                expect(kubernetesCommandLineToolsExecutor.waitDevWorkspace().stdout).contains('condition met');
+            });
 
-    test('Wait until DevWorkspace has status "ready"', async function (): Promise<void> {
-        expect(kubernetesCommandLineToolsExecutor.waitDevWorkspace().stdout).contains('condition met');
-    });
+            test('Check if project was created', function (): void {
+                clonedProjectName = StringUtil.getProjectNameFromGitUrl(devfileSample.link);
+                expect(containerTerminal.ls().stdout).includes(clonedProjectName);
+            });
 
-    suiteTeardown('Delete DevWorkspace', async function (): Promise<void> {
-        kubernetesCommandLineToolsExecutor.deleteDevWorkspace();
-    });
-});
+            test('Check if files were imported', function (): void {
+                containerWorkDir = containerTerminal.pwd().stdout.replace('\n', '');
+                expect(containerTerminal.ls(`${containerWorkDir}/${clonedProjectName}`).stdout).includes(`devfile.yaml`);
+            });
 
+            test(`Check if build command returns success`, function (): void {
```

Comment:
There could be several build commands in devfile.yaml, theoretically, but the test mentioned only one, which looks inconsistent, IMHO.

Reply:
Okay, I will change it for "commands"

Reply:
thanks